pyccl/tracers.py

```diff
@@ -24,6 +25,20 @@ def _Sig_MG(cosmo, a, k=None):
     return _vectorize_fn6(lib.Sig_MG, lib.Sig_MG_vec, cosmo, a, k)
 
 
+def _check_background_spline_compatability(cosmo, z):
```

Comment:
I think it should be "compatibility" ;-)

Reply:
Done.